server/src/main/scala/cromwell/CromwellEntryPoint.scala

```diff
@@ -44,6 +44,9 @@ object CromwellEntryPoint extends GracefulStopSupport {
 
   val gracefulShutdown = config.as[Boolean]("system.graceful-server-shutdown")
 
+  // 3 minute DNS TTL down from JVM default of infinite [BA-6454]
+  java.security.Security.setProperty("networkaddress.cache.ttl" , "180")
```

Comment:
```suggestion
val dnsCacheTtl = config.getOrElse("system.dns-cache-ttl", 3 seconds)
java.security.Security.setProperty("networkaddress.cache.ttl", dnsCacheTtl.toSeconds.toString)
```

Reply:
See my other comment, I have a suspicion that we could just put `networkaddress.cache.ttl` directly in our config file and skip the indirection